include/sender_cadmium.hpp

```diff
@@ -146,8 +150,8 @@ class Sender {
 
         // confluence transition
         void confluence_transition(
-		            TIME e,
-                            typename make_message_bags<input_ports>::type mbs) {
+            TIME e,
```

Comment:
Line breaking and intention issue: 
"make_message_bags<input_ports>::type mbs) { " --> this can go along with typename, it will be less than 80 characters.
i.e., typename make_message_bags<input_ports>::type mbs) {

Reply:
Resolved